tests/page/expect-timeout.spec.ts

```diff
@@ -20,25 +20,25 @@ import { test, expect } from './pageTest';
 test('should print timed out error message', async ({ page }) => {
   await page.setContent('<div id=node>Text content</div>');
   const error = await expect(page.locator('no-such-thing')).toHaveText('hey', { timeout: 1000 }).catch(e => e);
-  expect(stripAnsi(error.message)).toContain('Timed out 1000ms waiting for expect(received).toHaveText(expected)');
+  expect(stripAnsi(error.message)).toContain(`Timed out 1000ms waiting for expect(locator('no-such-thing')).toHaveText(hey)`);
```

Comment:
Should we quote "hey"?

Reply:
Not sure, in fact stock expect leaves those as `received` and `expected` which I think it is weird. I wonder if I should be saying `locator` instead of the actual value. Let me see how it would look.